packages/config/config/devices/0x028c/pwly-7828-a1.json

```diff
@@ -0,0 +1,287 @@
+{
+	"manufacturer": "Powerley",
+	"manufacturerId": "0x028c",
+	"label": "PWLY-7828-A1",
+	"description": "Thermostat",
+	"devices": [
+		{
+			"productType": "0xa55a",
+			"productId": "0x0001",
+			"zwaveAllianceId": 2401
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Display Units",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Celsius",
+					"value": 0
+				},
+				{
+					"label": "Fahrenheit",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "HVAC Settings",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "1 Stage Conventional",
+					"value": 0
+				},
+				{
+					"label": "2 Stage Conventional",
+					"value": 1
+				},
+				{
+					"label": "Heat Pump with O-Wire",
+					"value": 2
+				},
+				{
+					"label": "Heat Pump with B-Wire",
+					"value": 3
+				},
+				{
+					"label": "Heat Pump with O-Wire and Aux Heating",
+					"value": 4
+				},
+				{
+					"label": "Heat Pump with B-Wire and Aux Heating",
+					"value": 5
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "ECO Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Thermostat Default Settings",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "No Change",
+					"value": 0
+				},
+				{
+					"label": "Restore Thermostat Factory Settings (excluding z-wave settings)",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "5",
+			"label": "Temperature Calibration",
+			"description": "0-5: 0 - 5°F, 251-255: -5 - -1°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unit": "°F",
+			"unsigned": true,
```

Comment:
This is a signed parameter, with a range of -5..+5. 251...255 unsigned is -5..-1 when interpreted as signed.